Sources/UI/PaymentMethods/SepaDataView.swift

```diff
@@ -48,15 +48,33 @@ public struct CountryPicker: View {
     }
 }
 
+extension String {
+    var flag: String {
+        let base: UInt32 = 127397
+        var result = ""
+        for char in self.unicodeScalars {
+            result.unicodeScalars.append(UnicodeScalar(base + char.value)!)
+        }
+        return String(result)
+    }
```

Comment:
Was hältst du davon die Flag `String?` zu machen und damit das Force Unwrapping in Zeile 56 zu verhindern. Falles es mal für einen bestimmen Fall keine UnicodeScalar geben sollte können wir dennoch die UI anzeigen nur ohne Flagge anstatt das die App abstürzt.

Reply:
2a4a15d2e40c83fd1c97c9bbbe2af4f308bcaf94